src/transport/RendezvousSession.h

```diff
@@ -86,13 +91,25 @@ class RendezvousSession : public SecurePairingSessionDelegate, public Rendezvous
     void OnRendezvousError(CHIP_ERROR err) override;
     void OnRendezvousMessageReceived(PacketBuffer * buffer) override;
 
+    const IPAddress & GetIPAddress() const { return mDeviceAddress; }
```

Comment:
This will also benefit from some documentation, since it is not as clear what the IP address is (there is no setter.. the only setter-like is 'send ip address' but that does not seem to be it.

Could be done in a future PR.